packages/search-ui/src/__tests__/DebounceManager.test.js

```diff
@@ -0,0 +1,111 @@
+import DebounceManager from "../DebounceManager";
+
+describe("#runWithDebounce", () => {
+  it("will run the provided function after the provided number of milliseconds", () => {
+    const manager = new DebounceManager();
+    let debounced = jest.fn();
```

Comment:
I don't think we're overriding `debounced` after at any point, are we? Can this be `const debounced` instead of `let`?

Reply:
Same question as below for all other instances of `let debounced`

Reply:
Yes, good catch.